src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/MessageProcessor.java

```diff
@@ -22,47 +18,44 @@
     private static final Logger logger = LoggerFactory.getLogger(MessageProcessor.class);
 
     private final ReceiverProvider receiverProvider;
-    private final BulkScanProcessorClient bulkScanProcessorClient;
+    private final IMessageHandler envelopeProcessor;
 
-    public MessageProcessor(
-        ReceiverProvider receiverProvider,
-        BulkScanProcessorClient bulkScanProcessorClient
-    ) {
+    public MessageProcessor(ReceiverProvider receiverProvider, IMessageHandler envelopeProcessor) {
         this.receiverProvider = receiverProvider;
-        this.bulkScanProcessorClient = bulkScanProcessorClient;
+        this.envelopeProcessor = envelopeProcessor;
     }
 
     @Scheduled(fixedDelayString = "${queue.read-interval}")
     public void run() {
-        logger.info("Checking queue for new messages");
```

Comment:
is this related?

Reply:
It  will fill the log with "Checking queue for new messages" which is not particularly helpful. 
Maybe putting it as a debug message or removing it (I took this option) would be the most sensible.